{"name": "Try to update pet with invalid ID False", "status": "failed", "statusDetails": {"message": "AssertionError: Unexpected status code! Expected: 400. Actual: 500", "trace": "self = <tests.test_update_pet.TestUpdatePet object at 0x106306740>, invalid_id = False\n\n    @allure.severity(allure.severity_level.CRITICAL)\n    @allure.title(\"Try to update pet with invalid ID {invalid_id}\")\n    @pytest.mark.parametrize(\"invalid_id\", invalid_ids)\n    def test_update_pet_with_invalid_id(self, invalid_id):\n        data = {\n            \"id\": invalid_id,\n            \"name\": self.fake.first_name(),\n            \"photoUrls\": [\"string\"]\n        }\n    \n        response = MyRequests.put(\n            \"/pet/\",\n            data)\n    \n>       Assertions.assert_code_status(response, 400)\n\ntests/test_update_pet.py:107: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [500]>, expected_status_code = 400\n\n    @staticmethod\n    @allure.step(\"Check status code is {expected_status_code}\")\n    def assert_code_status(response: Response, expected_status_code):\n>       assert response.status_code == expected_status_code, \\\n            f\"Unexpected status code! Expected: {expected_status_code}. Actual: {response.status_code}\"\nE       AssertionError: Unexpected status code! Expected: 400. Actual: 500\n\nlib/assertions.py:30: AssertionError"}, "steps": [{"name": "PUT request to URL '/pet/'", "status": "passed", "start": 1744657923421, "stop": 1744657923431}, {"name": "Check status code is 400", "status": "failed", "statusDetails": {"message": "AssertionError: Unexpected status code! Expected: 400. Actual: 500\n", "trace": "  File \"/Users/miowch/.pyenv/versions/n26_nastia_kuzina/lib/python3.10/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/Users/miowch/Documents/GitHub/nastia-kuzina/Task3/api-e2e-tests/lib/assertions.py\", line 30, in assert_code_status\n    assert response.status_code == expected_status_code, \\\n"}, "parameters": [{"name": "response", "value": "<Response [500]>"}, {"name": "expected_status_code", "value": "400"}], "start": 1744657923431, "stop": 1744657923431}], "parameters": [{"name": "invalid_id", "value": "False"}], "start": 1744657923421, "stop": 1744657923431, "uuid": "2072fb90-450c-4810-ac0d-9a4ba59a301b", "historyId": "231fca04ba4983866a881c5d0c29f3ec", "testCaseId": "dad9c3a12b4ea201599875acfbfa599e", "fullName": "tests.test_update_pet.TestUpdatePet#test_update_pet_with_invalid_id", "labels": [{"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_update_pet"}, {"name": "subSuite", "value": "TestUpdatePet"}, {"name": "host", "value": "iMac.local"}, {"name": "thread", "value": "58784-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_update_pet"}]}